Formula/gum.rb

```diff
@@ -0,0 +1,32 @@
+class Gum < Formula
+  desc "Tool for glamorous shell scripts"
+  homepage "https://github.com/charmbracelet/gum"
+  url "https://github.com/charmbracelet/gum/archive/v0.1.0.tar.gz"
+  sha256 "f89228867cd61ebeae0b7bcc9e8034079da256e2c6bf696a9eceb73fc4c3f2b6"
+  license "MIT"
+
```

Comment:
Should a [`head` URL](https://docs.brew.sh/Formula-Cookbook#unstable-versions-head) be added?

```suggestion
  head "https://github.com/charmbracelet/gum.git", branch: "main"

```

Reply:
Yes! Good catch, thank you!